src/AppendStream.php

```diff
@@ -237,6 +237,11 @@ public function write($string): int
         throw new \RuntimeException('Cannot write to an AppendStream');
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @return array|mixed
```

Comment:
`array|mixed` doesn't make sense as a type. it's like writing `false|bool`. just write `mixed`.

Reply:
Agreed.